app/Belinska/test_task_unit_tests.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from app.task import Task
+
+
+@pytest.mark.priority_low
+@pytest.mark.parametrize("task_data", [
+    {"title": "Task 1", "description": "Description 1", "due_date": "28.07.2023", "priority": "Low"},
+    {"title": "Task 2", "description": "Description 2", "due_date": "29.07.2023", "priority": "Low"},
+    {"title": "Task 3", "description": "Description 3", "due_date": "30.07.2023", "priority": "Low"},
+])
+def test_task_low_specification(general_task, task_data):
+    assert general_task.title == task_data["title"]
+    assert general_task.description == task_data["description"]
+    assert general_task.due_date == task_data["due_date"]
+    assert general_task.priority == task_data["priority"]
+    assert not general_task.completed
+
+
+@pytest.mark.priority_high
+def test_task_high_specification(specific_task):
+    assert specific_task.title == "Specific Task Title"
+    assert specific_task.description == "This is a Specific task description"
+    assert specific_task.due_date == "29.07.2023"
+    assert specific_task.priority == "High"
+    assert not specific_task.completed
+
+
+def test_mark_as_completed(completed_task):
```

Comment:
'completed_task' must be a parameter or a fixture. However here it is not. So the test fails because it has an unexpected argument